library/spdm_common_lib/libspdm_com_crypto_service.c

```diff
@@ -1032,87 +1012,61 @@ bool libspdm_verify_challenge_auth_signature(libspdm_context_t *spdm_context,
             if (!result) {
```

Comment:
This check can be merged with the check at line 1019, and move to the line after 1022.

Reply:
agree